static/js/EditFilenameForm.js

```diff
@@ -0,0 +1,53 @@
+define('editFilenameForm', [ 'jquery', 'jquery-ui', 'underscore', 'RemoteStateChangeMonitor', 'tpl!../templates/admin/editFilenameForm', 
+		'ModalLoadingOverlay', 'AbstractFileUploadForm', 'AlertHandler'], 
+		function($, ui, _, RemoteStateChangeMonitor, editFilenameForm, ModalLoadingOverlay, AbstractFileUploadForm) {
+	
+	var defaultOptions = {
+			title : 'Edit Filename',
+			createFormTemplate : editFilenameForm,
+			showUploadProgress : false
+	};
+	
+	function editFilenameForm(options) {
```

Comment:
Since this is a class, the first letter should be capitalized, so EditFilenameForm